environment.rb

```diff
@@ -0,0 +1,101 @@
+module Environment
+	class Depot
```

Comment:
Indentation should be two spaces, be careful when setting the TAB range in your IDE, sometimes Git just ignore the setting and take a tab as a 4-spaces-TAB.